<ui version="4.0" >
 <class>SvnPropSetDialog</class>
 <widget class="QDialog" name="SvnPropSetDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Set Subversion Property</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="1" column="1" >
      <widget class="QTextEdit" name="propTextEdit" >
       <property name="toolTip" >
        <string>Enter text of the property</string>
       </property>
       <property name="tabChangesFocus" >
        <bool>true</bool>
       </property>
       <property name="acceptRichText" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>Property &amp;Name:</string>
       </property>
       <property name="buddy" >
        <cstring>propNameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="propNameEdit" >
       <property name="toolTip" >
        <string>Enter the name of the property to be set</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2" >
      <widget class="QCheckBox" name="recurseCheckBox" >
       <property name="toolTip" >
        <string>Select to apply the property recursively</string>
       </property>
       <property name="text" >
        <string>Apply &amp;recursively</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Property &amp;Value:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="buddy" >
        <cstring>propTextEdit</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>propNameEdit</tabstop>
  <tabstop>propTextEdit</tabstop>
  <tabstop>recurseCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SvnPropSetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>28</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel" >
     <x>28</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SvnPropSetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>94</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel" >
     <x>94</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
